Makefile

```diff
@@ -469,17 +472,17 @@ $(CSS_DEST): node_modules $(CSS_SOURCES)
 
 .PHONY: javascripts
 javascripts:
-	echo "'make javascripts' is deprecated, please use 'make js'"
+	echo "'$(MAKE) javascripts' is deprecated, please use 'make js'"
```

Comment:
still have `make js`

Reply:
Ah man. Thanks. I'll fix that right away.